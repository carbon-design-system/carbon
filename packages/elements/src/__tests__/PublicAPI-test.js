/**
 * Copyright IBM Corp. 2018, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import * as CarbonElements from '../';

test('Public API should only change with a semver change', () => {
  expect(Object.keys(CarbonElements).sort()).toMatchSnapshot();
});
