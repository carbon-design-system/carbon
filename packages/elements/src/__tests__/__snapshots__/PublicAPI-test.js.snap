// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Public API should only change with a semver change 1`] = `
Array [
  "active01",
  "activeDanger",
  "activeLightUI",
  "activePrimary",
  "activeSecondary",
  "activeTertiary",
  "activeUI",
  "baseFontSize",
  "black",
  "black100",
  "blue",
  "blue10",
  "blue100",
  "blue20",
  "blue30",
  "blue40",
  "blue50",
  "blue60",
  "blue70",
  "blue80",
  "blue90",
  "bodyLong01",
  "bodyLong02",
  "bodyShort01",
  "bodyShort02",
  "brand01",
  "brand02",
  "brand03",
  "breakpoint",
  "breakpointDown",
  "breakpointUp",
  "breakpoints",
  "buttonSeparator",
  "caption01",
  "code01",
  "code02",
  "colors",
  "container",
  "container01",
  "container02",
  "container03",
  "container04",
  "container05",
  "coolGray",
  "coolGray10",
  "coolGray100",
  "coolGray20",
  "coolGray30",
  "coolGray40",
  "coolGray50",
  "coolGray60",
  "coolGray70",
  "coolGray80",
  "coolGray90",
  "cyan",
  "cyan10",
  "cyan100",
  "cyan20",
  "cyan30",
  "cyan40",
  "cyan50",
  "cyan60",
  "cyan70",
  "cyan80",
  "cyan90",
  "danger",
  "danger01",
  "danger02",
  "decorative01",
  "disabled01",
  "disabled02",
  "disabled03",
  "display01",
  "display02",
  "display03",
  "display04",
  "easings",
  "em",
  "expressiveHeading01",
  "expressiveHeading02",
  "expressiveHeading03",
  "expressiveHeading04",
  "expressiveHeading05",
  "expressiveHeading06",
  "expressiveParagraph01",
  "fast01",
  "fast02",
  "field01",
  "field02",
  "fluid",
  "fluidSpacing",
  "fluidSpacing01",
  "fluidSpacing02",
  "fluidSpacing03",
  "fluidSpacing04",
  "focus",
  "fontFamilies",
  "fontFamily",
  "fontWeight",
  "fontWeights",
  "formatTokenName",
  "g10",
  "g100",
  "g90",
  "getTypeSize",
  "gray",
  "gray10",
  "gray100",
  "gray20",
  "gray30",
  "gray40",
  "gray50",
  "gray60",
  "gray70",
  "gray80",
  "gray90",
  "green",
  "green10",
  "green100",
  "green20",
  "green30",
  "green40",
  "green50",
  "green60",
  "green70",
  "green80",
  "green90",
  "heading01",
  "heading02",
  "helperText01",
  "highlight",
  "hoverDanger",
  "hoverField",
  "hoverLightUI",
  "hoverPrimary",
  "hoverPrimaryText",
  "hoverRow",
  "hoverSecondary",
  "hoverSelectedUI",
  "hoverTertiary",
  "hoverUI",
  "icon01",
  "icon02",
  "icon03",
  "iconSize",
  "iconSize01",
  "iconSize02",
  "interactive01",
  "interactive02",
  "interactive03",
  "interactive04",
  "inverse01",
  "inverse02",
  "inverseFocusUi",
  "inverseHoverUI",
  "inverseLink",
  "inverseSupport01",
  "inverseSupport02",
  "inverseSupport03",
  "inverseSupport04",
  "label01",
  "layout",
  "layout01",
  "layout02",
  "layout03",
  "layout04",
  "layout05",
  "layout06",
  "layout07",
  "link01",
  "link02",
  "magenta",
  "magenta10",
  "magenta100",
  "magenta20",
  "magenta30",
  "magenta40",
  "magenta50",
  "magenta60",
  "magenta70",
  "magenta80",
  "magenta90",
  "miniUnit",
  "miniUnits",
  "moderate01",
  "moderate02",
  "motion",
  "orange",
  "orange40",
  "orange60",
  "orange70",
  "overlay01",
  "print",
  "productiveHeading01",
  "productiveHeading02",
  "productiveHeading03",
  "productiveHeading04",
  "productiveHeading05",
  "productiveHeading06",
  "productiveHeading07",
  "purple",
  "purple10",
  "purple100",
  "purple20",
  "purple30",
  "purple40",
  "purple50",
  "purple60",
  "purple70",
  "purple80",
  "purple90",
  "px",
  "quotation01",
  "quotation02",
  "red",
  "red10",
  "red100",
  "red20",
  "red30",
  "red40",
  "red50",
  "red60",
  "red70",
  "red80",
  "red90",
  "rem",
  "reset",
  "rgba",
  "scale",
  "selectedLightUI",
  "selectedUI",
  "skeleton01",
  "skeleton02",
  "slow01",
  "slow02",
  "spacing",
  "spacing01",
  "spacing02",
  "spacing03",
  "spacing04",
  "spacing05",
  "spacing06",
  "spacing07",
  "spacing08",
  "spacing09",
  "spacing10",
  "spacing11",
  "spacing12",
  "styles",
  "support01",
  "support02",
  "support03",
  "support04",
  "teal",
  "teal10",
  "teal100",
  "teal20",
  "teal30",
  "teal40",
  "teal50",
  "teal60",
  "teal70",
  "teal80",
  "teal90",
  "text01",
  "text02",
  "text03",
  "text04",
  "text05",
  "textError",
  "themes",
  "tokens",
  "ui01",
  "ui02",
  "ui03",
  "ui04",
  "ui05",
  "uiBackground",
  "unstable__meta",
  "unstable_tokens",
  "v9",
  "visitedLink",
  "warmGray",
  "warmGray10",
  "warmGray100",
  "warmGray20",
  "warmGray30",
  "warmGray40",
  "warmGray50",
  "warmGray60",
  "warmGray70",
  "warmGray80",
  "warmGray90",
  "white",
  "white0",
  "yellow",
  "yellow20",
  "yellow30",
  "yellow40",
  "yellow50",
]
`;
